applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -139,18 +180,31 @@ def Initialize(self):
         self.response_function_utility.Initialize()
 
     def CalculateValue(self):
+        Logger.PrintInfo("\n> Starting primal analysis for response:", self.identifier)
```

Comment:
I think this prints two `:` =>Starting primal analysis for response: :", self.identifier